Ref/SignalGen/SignalGen.fpp

```diff
@@ -93,6 +93,12 @@ module Ref {
 
     include "Events.fppi"
 
+    # ----------------------------------------------------------------------
+    # Interfaces
+    # ----------------------------------------------------------------------
+
+    include "../../Fw\Interfaces\ChannelInterfaces.fppi"
```

Comment:
You should use `/` instead of `\` as well.

Reply:
Should the channel interface be used for all components in Ref and RPI as well? Or are we happy to keep it hybrid?

Reply:
Not just yet.  Let's get it in and proven in SignalGen, then we can slowly refactor.